scripts/score-project-quality.py

```
@@ -67,6 +69,71 @@ def to_int_or_none(s):
+    """法人番号の公式名（houjin.db）。未登録/DB無しは None。CN単位でキャッシュ。"""
+    if _houjin_cur is None:
+        return None
+    if cn in _houjin_name_cache:
+        return _houjin_name_cache[cn]
+    row = _houjin_cur.execute('SELECT name FROM houjin WHERE corporate_number=?', (cn,)).fetchone()
+    name = row[0] if row else None
+    _houjin_name_cache[cn] = name
+    return name
+
+def is_cn_verified(cn: str, recipient_name: str) -> bool:
+    """有効な法人番号 かつ houjin.db 公式名が支出先名と一致するか（正規化名で照合）。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)